test/unit/collection_test.rb

```diff
@@ -60,6 +60,66 @@ def test_xpath_item_locator__matches_exact_text
       assert_equal ".//div[./text() = 'Example Text']", magazine.send(:item_xpath)
     end
 
+    def test_locator_options__blank
+      bullets = Class.new(AePageObjects::Element)
+      clip    = Class.new(AePageObjects::Collection) do
+        self.item_class = bullets
+      end
+
+      magazine_node = mock
+      parent_node = mock(find: magazine_node)
+      parent = mock(node: parent_node)
+
+      magazine = clip.new(parent, :name => "18_holder", :item_locator => ".some_class")
+      magazine.stubs(:item_xpath).returns('item_xpath')
+
+      bullet1_stub = mock
+      magazine_node.expects(:all).with(:xpath, 'item_xpath', {}).returns([bullet1_stub])
+      magazine_node.expects(:find).with(:xpath, 'item_xpath[1]', {}).returns(bullet1_stub)
+      assert_equal bullet1_stub, magazine.at(0).node
+    end
+
+    def test_locator_options__present
+      bullets = Class.new(AePageObjects::Element)
+      clip    = Class.new(AePageObjects::Collection) do
+        self.item_class = bullets
+      end
+
+      magazine_node = mock
+      parent_node = mock(find: magazine_node)
+      parent = mock(node: parent_node)
+
+      magazine = clip.new(parent, :name => "18_holder", :item_locator => [".some_class", { capybara: 'options' }])
+      magazine.stubs(:item_xpath).returns("item_xpath")
+
+      bullet1_stub = mock
+      magazine_node.expects(:all).with(:xpath, 'item_xpath', { capybara: 'options' }).returns([bullet1_stub])
+      magazine_node.expects(:find).with(:xpath, "item_xpath[1]", { capybara: 'options' }).returns(bullet1_stub)
+      assert_equal bullet1_stub, magazine.at(0).node
+    end
+
+    def test_locator_options__exact_false
```

Comment:
Tim, I'm sorry that moving the test was an inconvenience, but your attitude is not appreciated. You are not the only one with other work to do. I've merged the PR.